src/integration/checkout/services/checkout.service.ts

```diff
@@ -6,6 +6,12 @@ import { Fiat } from 'src/shared/models/fiat/fiat.entity';
 import { Language } from 'src/shared/models/language/language.entity';
 import { CheckoutHostedPayment, CheckoutLanguages, CheckoutPagedResponse, CheckoutPayment } from '../dto/checkout.dto';
 
+interface CheckoutBalance {
+  balances: { available: number; collateral: number; payable: number; pending: number };
```

Comment:
Maybe create separate balance interface for `available: number; collateral: number; payable: number; pending: number` because it's used in CheckoutData in the observer too